pkg/trivyk8s/trivyk8s_test.go

```diff
@@ -345,3 +345,117 @@ func TestInitResources(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
While it's good to test this function in isolation, I think there's more value in testing `ListSpecificArtifacts` instead as the functionality lies in there. As we already saw with the bug that you just fixed, this test did not pick it up as `ListSpecificArtifacts` was untested.

Reply:
as you know I agree with you about tests, so I've added a new way for testing #448
it covers `ListSpecificArtificats` at first, but we can use it for other methods.
Wdyt?